src/theme.rs

```diff
@@ -123,13 +102,10 @@ impl Theme {
 
             let background = transpose(background.map(|fg| self.get_color(fg)))?;
 
-            let mut style: String = format!("{font_style}", font_style = *font_style_ansi,);
+            let mut style: String = format!("{font_style}");
```

Comment:
This is not quite what we had before, right? If there is no foreground color and no background color, there will now be a trailing semicolon.

Reply:
Ah, I believe the tests are also failing because of this.

Reply:
I believe I have fixed this.